app/Domain/Integrations/Events/IntegrationActivatedWithCoupon.php

```diff
@@ -7,6 +7,7 @@
 use Illuminate\Foundation\Events\Dispatchable;
 use Ramsey\Uuid\UuidInterface;
 
+// @deprecated
```

Comment:
I think the @deprecated tags need to be in full PHPDocs to be properly recognized

![Screenshot 2024-03-21 at 14 24 04](https://github.com/cultuurnet/publiq-platform/assets/1321596/d03eca8a-c758-4132-9811-7039cf77d8dd)

Reply:
Good to know! Luckily the next step is to manually remove all that `@deprecated` code, so I will not change it 😏